packages/sprite/src/Sprite.js

```diff
@@ -590,3 +554,10 @@ export default class Sprite extends Container
         }
     }
 }
+
+Sprite.fromImage = Sprite.from;
+Sprite.fromSVG = Sprite.from;
+Sprite.fromCanvas = Sprite.from;
+Sprite.fromVideo = Sprite.from;
+Sprite.fromFrame = Sprite.from;
```

Comment:
We should definitely deprecate these. Particularly converting `fromImage` and `fromSVG` function signatures to match `from(source, options)`. @GoodBoyDigital, do you want me to take care of the deprecations in this PR?

Reply:
Lets bury the stewardess

Reply:
Hey @bigtimebuddy if you could that would be great! I think they also exist like that in the Texture class too.